qt/search_panel.cpp

```diff
@@ -188,7 +188,7 @@ bool SearchPanel::Try3dModeCmd(std::string const & str)
   if (!is3dModeOn && !is3dBuildingsOn && !is3dModeOff)
     return false;
 
-  GetFramework().Allow3dMode(is3dModeOn || is3dBuildingsOn, is3dBuildingsOn);
+  GetFramework().Allow3dMode(false, is3dModeOn, is3dBuildingsOn);
```

Comment:
Add a switch for the "3d mode always"?

Also is it possible to have a 3d mode without 3d buildings?

Reply:
> Add a switch for the "3d mode always"?

IMHO these switches here are all hacks, and perhaps they should be rather removed completely. IMHO, the UI elements on the preferences dialog should be used exclusively.

> Also is it possible to have a 3d mode without 3d buildings?

Yes absolutely possible. You can try it out on the preferences dialog.